data/sidk.dat

```diff
@@ -0,0 +1 @@
+19.051741, -155.807678, ,
```

Comment:
```suggestion
19.051741, -155.807678, blue , 5
```

Please, add a color and a number from 0 to 10. I put a suggestion, but you can choose something else.